compliance-report-os-is-vendor-supported/os-is-vendor-supported.json

```diff
@@ -98,14 +98,14 @@
     "supported-centos": {
       "id": "supported-centos",
       "name": "Supported CentOS",
-      "description": "CentOS hosts are running version 7 or 8",
+      "description": "CentOS hosts are running Stream 8",
```

Comment:
I suppose we would need to pull in another attribute here to get specific, but really only the `sys.os_release` variable has "stream" in it: 

```
{"ANSI_COLOR":"0;31","BUG_REPORT_URL":"https:\/\/bugzilla.redhat.com\/","CPE_NAME":"cpe:\/o:centos:centos:8","HOME_URL":"https:\/\/centos.org\/","ID":"centos","ID_LIKE":"rhel fedora","NAME":"CentOS Stream","PLATFORM_ID":"platform:el8","PRETTY_NAME":"CentOS Stream 8","REDHAT_SUPPORT_PRODUCT":"Red Hat Enterprise Linux 8","REDHAT_SUPPORT_PRODUCT_VERSION":"CentOS Stream","VERSION":"8","VERSION_ID":"8"}
```